docs/api/Bs.Dyn.html

```diff
@@ -205,7 +205,7 @@
 </div>
 
 <div class="type-declaration">
-<pre><span id="TYPEto_value"><span class="keyword">type</span> <code class="type">'a</code> to_value</span> = <code class="type">('a -> <a href="Bs_dyn.html#TYPEvalue">value</a> [@bs])</code> </pre>
+<pre><span id="TYPEto_value"><span class="keyword">type</span> <code class="type">'a</code> to_value</span> = <code class="type">([ `Arity_1 of 'a ], <a href="Bs_dyn.html#TYPEvalue">value</a>) Js.Internal.fn</code> </pre>
```

Comment:
it seems you are not using our patched compiler - 4.02.3+bs-master
here the docs is no longer `'a -> 'b [@bs]`

Reply:
I went through and tried this again. Perhaps you can tell me what might be going on.
```
$ opam update
$ opam switch 4.02.3+buckle-master
$ eval `opam config env`
$ cd $HOME/src/bucklescript
$ npm install
$ cd jscomp
$ make world
$ make docs
```
The `make docs` output is:
```
make -C bin bsppx.exe
make[1]: Entering directory '/home/zach/src/bucklescript/jscomp/bin'
make[1]: 'bsppx.exe' is up to date.
make[1]: Leaving directory '/home/zach/src/bucklescript/jscomp/bin'
node ../scripts/doc_gen.js
Running ocamldoc.opt -g /home/zach/src/bucklescript/odoc_gen/generator.cmxs  -w -40 -nostdlib -I /home/zach/src/bucklescript/jscomp/stdlib -I /home/zach/src/bucklescript/jscomp/others -I /home/zach/src/bucklescript/jscomp/runtime -charset utf-8  -intro /home/zach/src/bucklescript/docs/api/intro.txt -sort -ppx /home/zach/src/bucklescript/jscomp/bin/bsppx.exe  -d /home/zach/src/bucklescript/docs/api  runtime/js.ml runtime/js.mli runtime/js_exn.ml runtime/js_exn.mli runtime/js_float.ml runtime/js_int.ml runtime/js_int64.ml runtime/js_internal.ml runtime/js_internal.mli runtime/js_nativeint.ml runtime/js_null.ml runtime/js_null.mli runtime/js_primitive.ml runtime/js_primitive.mli runtime/js_typed_array.ml runtime/js_undefined.ml runtime/js_undefined.mli runtime/js_unsafe.ml runtime/js_unsafe.mli others/bs.ml others/bs_dyn.ml others/bs_dyn.mli others/bs_dyn_lib.ml others/bs_dyn_lib.mli others/dom.mli others/dom_storage.mli others/js_array.ml others/js_boolean.ml others/js_boolean.mli others/js_cast.ml others/js_cast.mli others/js_date.ml others/js_dict.ml others/js_dict.mli others/js_global.ml others/js_json.ml others/js_json.mli others/js_list.ml others/js_list.mli others/js_math.ml others/js_null_undefined.ml others/js_null_undefined.mli others/js_obj.ml others/js_option.ml others/js_option.mli others/js_promise.ml others/js_re.mli others/js_result.ml others/js_result.mli others/js_string.ml others/js_types.ml others/js_types.mli others/js_vector.ml others/js_vector.mli others/node.ml others/node_buffer.ml others/node_child_process.ml others/node_fs.ml others/node_module.ml others/node_path.ml others/node_process.ml others/node_process.mli
Warning: Element Js.Null_undefined.t not found
```
This does not produce any differences from the API docs currently submitted in the pull request. Is the 4.02.3+buckle-master not the patched compiler?

I tried again following the instructions under "Install from Source" -> "Minimal Dependencies" where I manually built the OCaml compiler and this seems to have worked. I will update the pull request.